extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -107,17 +107,20 @@ public static function onInstantGlobalsGetNewsAndStoriesVariables( array &$vars
 		$vars[] = 'wgAdDriverPorvataMoatTrackingCountries';
 		$vars[] = 'wgAdDriverPorvataMoatTrackingSampling';
 
-		// news&stories variables only
-		$vars[] = 'wgAdDriverInstartLogicF2RecoveryCountries';
+		// TODO: Remove after src=[gpt,ns] is finished
 		$vars[] = 'wgAdDriverNewsAndStoriesSingleSrcKeyValueCountries';
 		$vars[] = 'wgAdDriverNewsAndStoriesSrcKeyValueCountries';
+		// TODO: Remove after ADEN-6155
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app